R/JASPDescriptivesPlot.R

```diff
@@ -0,0 +1,105 @@
+
+#' Descriptive plot of categorical variable (x-axis) vs. a continuous variable (y-axis)
+#'
+#' @param x character vector indicating the groups shown on the x-axis.
+#' @param y numeric vector indicating the y-values.
+#' @param ciLower numeric vector with the lower bound of the confidence interval.
+#' @param ciUpper numeric vector with the upper bound of the confidence interval.
+#' @param xName character, name to use as the x-axis title.
+#' @param yName character, name to use as the y-axis title.
+#' @param group character vector indicating group membership, which is used for shape and fill color
+#' @param groupName character, name to show as the legend title.
+#' @param horizontalAsymptote numeric, draw a horizontal asymptote at this y-value.
+#' @param horizontalAsymptoteLineType line type of the horizontal asymptote.
+#' @param position a value for the position argument of e.g., geom_line.
+#' @param lineSize a value for the size argument of geom_line.
+#' @param pointSize a value for the size argument of geom_point.
+#' @param errorbarWidth a value for the width argument of geom_errorbar.
+#' @param hideXLevelNames Logical, should the level names on the x-axis be hidden?
```

Comment:
We had a discussion with @AlexanderLyNL regarding code style whether it makes sense for logical arguments be a verb. I think it can be quite natural in examples like these, but we did end up forbidding using verbs for arguments (https://github.com/jasp-stats/jasp-desktop/blob/stable/Docs/development/guide-option-names.md#refrain-from-using-verbs-for-option-names). Now this document is concerned with the QML file option names, but I think it makes sense to be consistent. 

I am happy to open this discussion again, or resolve it by not using the verb formulation here...

Reply:
I'll change it but I think this style decision is overly pedantic. If the style guide is not intuitive and reasonably simple to follow, nobody is going to follow it. This goes for `hideXLevelNames`, `showBreaksAtExtremaOnly`, and `doNotConnectPoints`, which I'll rename to `xLevelNames`, `breaksAtExtremaOnly`, and `connectedPoints`. I find these a lot less readable, though. Users will have no idea that they are supposed to put logical values in there without reading the documentation first.

Reply:
> Users will have no idea that they are supposed to put logical values in there without reading the documentation first.

I agree with you on this :) @AlexanderLyNL , do you want to weigh in?